packages/cli/src/index.ts

```diff
@@ -83,20 +83,20 @@ const args: ClientOpts = yargs(hideBin(process.argv))
     array: true,
     optional: true,
   })
-  .option('radiusHistory', {
-    describe: `2^r radius for history network client`,
+  .option('storageHistory', {
+    describe: `Storage space allocated to HistoryNetwork DB in MB`,
     number: true,
-    default: 256,
+    default: 1024,
   })
-  .option('radiusBeacon', {
-    describe: `2^r radius for beacon network client`,
+  .option('storageBeacon', {
+    describe: `Storage space allocated to BeaconChainNetwork DB in MB`,
     number: true,
-    default: 256,
+    default: 1024,
```

Comment:
Beacon Network shouldn't need anywhere near a GB. It's just the bootstrap, a few light client updates, and two headers.